Formula/aws-sso-cli.rb

```diff
@@ -0,0 +1,24 @@
+class AwsSsoCli < Formula
+  desc "Securely manage AWS API credentials using AWS SSO"
+  homepage "https://github.com/synfinatic/aws-sso-cli"
+  url "https://github.com/synfinatic/aws-sso-cli/archive/refs/tags/v1.9.9.tar.gz"
+  sha256 "ef3592a2fdf9d1812059dcbc7b397a110cb4de523f2ca8dec57a9d7a6bad54ef"
+  license "GPL-3.0-only"
+
+  depends_on "go" => [:build, :test]
+  depends_on xcode: [:build, :test]
+
+  def install
+    ENV["BREW_INSTALL"] = "1"
+    ENV["PROJECT_COMMIT"] = "2e15b09ca75285bfe7d8c8804a6f4eb9d7579482"
+
+    system "make"
+    bin.install "./dist/aws-sso"
```

Comment:
No `make install` target in upstream's Makefile?

Reply:
Yes, it honors `$INSTALL_PREFIX`.  wasn't clear from the docs if this was honored or needed to be set manually?